data/campaigns/Sceptre_of_Fire/scenarios/9_Caverns_of_Flame.cfg

```diff
@@ -672,15 +671,227 @@
+                    [/message]
+                [/else]
+            [/if]
+        [/event]
+        [event]
+            name=moveto
+            [filter]
+                x,y=29,31
+            [/filter]
+            [filter_condition]
+                [not]
+                    [variable]
+                        name=gathor_tablet
+                        equals=whole
+                    [/variable]
+                    [or]
+                        [have_unit]
+                            id=Toomak
+                        [/have_unit]
+                    [/or]
+                [/not]
+                [and]
+                    [variable]
+                       name=unit.id
+                       not_equals=Krawg
+                    [/variable]
+                [/and]
+            [/filter_condition]
             [message]
-                speaker=Krawg
-                message= _ "Wha’?"
+                speaker=$unit.id
+                message= _ "There’s a tablet with a rune here!"
+            [/message]
+            [message]
+                speaker=Thursagan
+                message= _ "Yes, that must be the last rune sealing the volcano. Smash it, like the others!"
+            [/message]
+            [message]
+                speaker=$unit.id
+                message= _ "The resulting eruption will lead to all of your deaths!"
+            [/message]
+            [message]
+                speaker=Thursagan
+                message= _ "Just do it! You riders have a chance to escape, our enemies cannot. We knew that death was likely when we fled here, just go!"
+            [/message]
+            [message]
+                speaker=Rugnur
+                message= _ "Wait! If the riders can escape, shouldn’t we give them the Sceptre?"
+            [/message]
+            [message]
+                speaker=Thursagan
+                message= _ "No, too risky, even if they drop it in King Haldric’s hand. You’ve seen how the elves have behaved, you’ve seen how your lord behaved. The corrupting emissions from the ruby are nothing compared to the greed already out there, and my crafted gold cage does nothing to contain that."
+            [/message]
+            [message]
+                speaker=Thursagan
+                message= _ "Durstorn and Glildur have both convinced me that the time is not right for the Sceptre of Fire, best that it rests with us for a while. Now, smash it!"
             [/message]
+            [message]
+                speaker=$unit.id
+                message= _ "Alright, there’s no turning back now!"
+            [/message]
+            [remove_item]
+                x,y=29,31
+                image=items/stone-tablet.png
+            [/remove_item]
+            [floating_text]
+                x,y=29,31
+                text="<span color='#ff7722'>" + _ "Smash!" + "</span>"
```

Comment:
Assuming I've understood the meaning correctly, `# po: Treat this as if it was a sound-effect in a comic book`